spec/requests/api/v1/course_spec.rb

```diff
@@ -28,14 +28,14 @@ def get_course_codes_from_body(body)
     context "without changed_since parameter" do
       before do
         site = FactoryBot.create(:site, code: "-", location_name: "Main Site", provider: provider)
-        subject1 = FactoryBot.find_or_create(:ucas_subject, :secondary)
-        subject2 = FactoryBot.find_or_create(:ucas_subject, :mathematics)
+        subject1 = FactoryBot.find_or_create(:subject, :modern_languages)
+        subject2 = FactoryBot.find_or_create(:subject, :german)
 
         course = FactoryBot.create(:course,
```

Comment:
That is how a valid ml course should look like

Reply:
There shouldn't be any need to do `FactoryBot.`